services/core-api/app/api/now_applications/models/now_application.py

```diff
@@ -59,6 +59,7 @@ class NOWApplication(Base, AuditMixin):
         db.String,
         db.ForeignKey('now_application_status.now_application_status_code'),
         nullable=False)
+    previous_application_status_code = db.Column(db.String)
```

Comment:
When the FK is made be sure to recognize it at the model level here

Reply:
updated.